scout/adapter/mongo/clinvar.py

```diff
@@ -460,16 +460,12 @@ def case_to_clinvars(self, case_id: str) -> Dict[str, dict]:
 
         return submitted_vars
 
-    def clinvar_cases(self, institute_id=None):
-        """Fetch all cases with variants contained in a ClinVar submission object
-
-        Args:
-            institute_id(str): id of an institute
-        Returns:
-            clinvar_case_ids(list): list of case _ids
-        """
+    def clinvar_cases(self, institute_id: str = None) -> List[str]:
+        """Fetch all cases with variants contained in a ClinVar submission object"""
         clinvar_case_ids = set()
-        inst_submissions = self.clinvar_submissions(institute_id=institute_id)
```

Comment:
This function doesn't exist anymore. It was split in 2, according to the type of submission